L126-core-add-alts-google-call-credentials.md

```diff
@@ -1,61 +1,74 @@
-# L126: C-core: Support ALTS Hard Bound Call Credentials in Google Default Credentials
+## L126: C-core: Support ALTS Hard Bound Call Credentials in Google Default Credentials
 
 *   Author(s): anniefrchz
 *   Approver: markdroth
 *   Status: In Review
 *   Implemented in: C++
 *   Last updated: 2025-08-22
-*   Discussion at:
+*   Discussion at:  
```

Comment:
A lot of lines seem to have gotten whitespace added to the end of the line.  Please remove.